Source/WebKitLegacy/mac/DOM/DOMRange.mm

```diff
@@ -236,10 +236,7 @@ - (DOMDocumentFragment *)createContextualFragment:(NSString *)html
 
 - (short)compareNode:(DOMNode *)refNode
 {
-    WebCore::JSMainThreadNullState state;
-    if (!refNode)
-        raiseTypeErrorException();
-    return raiseOnDOMError(IMPL->compareNode(*core(refNode)));
+    return NULL;
```

Comment:
Ditto. This should return 0, not NULL.

Reply:
Sure, will update. :-)

Reply:
Done in [3183fdb](https://github.com/WebKit/WebKit/commit/3183fdb64fb2abafc508f458ba691dbebaf567c9).